finality-verifiers/grandpa/src/bridges/runtime/storage_proof.rs

```diff
@@ -67,10 +67,10 @@ pub enum Error {
 /// Return valid storage proof and state root.
 ///
 /// NOTE: This should only be used for **testing**.
-#[cfg(feature = "testing")]
+#[cfg(feature = "std")]
```

Comment:
Why did we unhide this?

Reply:
that should have been there since build/v0.9.27 update. 

I change the feature to std since it uses other std features from sp_state_machine and it didn't build for parachains runtime before